cltk/tests/test_phonology.py

```diff
@@ -425,7 +426,22 @@ def test_arabic_transliterate(self):
         # from iso233-2 to arabic native script
         reverse = False
         assert AarabicTransliterate(mode,iso2332_string,ignore,reverse) == 'بِسْمِ اللهِ الرَّحْمٰنِ الرَّحِيْمِ'
+        
+    def test_middle_english_syllabify(self):
+        """Test syllabification for middle english"""
+        
+        words = ['marchall', 'content', 'thyne', 'greef', 'commaundyd']
 
+        syllabified = [word_me(w).syllabify() for w in words]
+        target_syllabified = [['mar', 'chall'], ['con', 'tent'], ['thyne'], ['greef'], ['com', 'mau', 'ndyd']]
+
+        assert syllabified == target_syllabified
+
+        syllabified_str = [word_me(w).syllabified_str() for w in words]
+        target_syllabified_str = ['mar.chall', 'con.tent', 'thyne', 'greef', 'com.mau.ndyd']
```

Comment:
I'm not sure I understand the value of the "." syllabified words, could you clarify in a comment in the class?
Is this to help with displaying the syllabification? if so, maybe users may want to be able to specify the separator character.

Reply:
I was wondering the same thing. And I like the idea of making it an option (with a default)

Reply:
As to the value, I've found it useful in my own syllabification library for Greek. If you want to display the syllabification to humans, you end up generally wanting to do a join on the syllable list with some delimiter. But, as you say, not necessarily ".", only be default.

Reply:
Actually, on further review, looks like this has been done.